path_roles_test.go

```diff
@@ -28,12 +28,11 @@ import (
 
 	"github.com/fatih/structs"
 	"github.com/go-test/deep"
-	"github.com/hashicorp/errwrap"
 	"github.com/hashicorp/vault/sdk/helper/certutil"
 	"github.com/hashicorp/vault/sdk/helper/strutil"
 	"github.com/hashicorp/vault/sdk/logical"
 
-	logicaltest "github.com/hashicorp/vault/helper/testhelpers/logical"
+	//logicaltest "github.com/hashicorp/vault/helper/testhelpers/logical"
```

Comment:
Can we please remove the commented code.

Reply:
removed